tools-reference/text.xml

```diff
@@ -9,6 +9,12 @@ This section provides an overview of various useful standard Unix and Gentoo
 tools and utilities that may be used within ebuilds or when working with
 ebuilds.
 </p>
+
+<p>
+Please keep in mind the need for
+<uri link="::ebuild-writing/error-handling/#the-die-function">error handling</uri>
```

Comment:
We use source references, not translated references in fragments (so if we change the translation in the XSL stylesheet, we don't have to update all links in the document). Therefore:
```suggestion
<uri link="::ebuild-writing/error-handling/#The die Function">error handling</uri>
```